evap/settings.py

```diff
@@ -59,6 +59,11 @@
 
 INTERNAL_USERNAMES_MAX_LENGTH = 20
 
+# List of tuples defining email domains that should be replaced when using OpenID login.
```

Comment:
The "when using OpenID" part might be misleading - if I understood it correctly, the current code will replace the email domain on a model level, so this will happen whenever a UserProfile is saved, no matter whether OpenID login is used or not.